src/components/Slider/Slider.tsx

```diff
@@ -84,8 +84,8 @@ export const BoemlySlider: React.FC<SliderProps> = ({
         mt="8"
       >
         <Flex flexDir="row" alignItems="center" justifyContent="center" width="full">
-          <Slider role="slider" width="10%" mr={2} cursor="default">
-            <SliderTrack backgroundColor="gray.200">
+          <Slider width="10%" mr={2} cursor="default">
+            <SliderTrack className="sliderTrack" backgroundColor="gray.200">
               <SliderFilledTrack backgroundColor="gray.200" />
             </SliderTrack>
           </Slider>
```

Comment:
```suggestion
          <Box width="10%" mr={2} backgroundColor="gray.200" borderRadius="full" height="..." />
```

I think rendering a slider here is not good - there are accessibility issues. E.g. if a user can only use the keyboard to navigate the page all those sliders would be taken into account and this might be confusing for this user.